woocommerce/woocommerce-ios
# Order Fulfillment: make customer note multiline

Fixes #1240 

### To test
1. Create an order with a customer note inside it. Make the customer note a few sentences long.
2. In the app, navigate to the order at select the "Begin Fulfillment" button to see the Fulfillment screen.

<img src="https://user-images.githubusercontent.com/1062444/70479831-d09a8380-1aa3-11ea-8e24-d773fe408c69.png" width="350" />

Update release notes:
